extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -102,6 +103,7 @@ require([
 	require.optional('ext.wikia.adEngine.slot.revcontentSlots')
 ], function (
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 11 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)